diff --git a/index.js b/index.js
index b94ea14b602befb622e376d5e1cd46819baa5d6c..97c9bf9f356771699df8382675bca4ac0fa9680c 100644
--- a/index.js
+++ b/index.js
@@ -1,4 +1,4 @@
-const Minipass = require('minipass')
+const {Minipass} = require('minipass')
 const EE = require('events')
 const isStream = s => s && s instanceof EE && (
   typeof s.pipe === 'function' || // readable
diff --git a/package.json b/package.json
index d608dc66ce515d3e6dd437b7fc029c519a99bd5a..9a684d5df8ea2b4bd39710d17c5832dc3250ad64 100644
--- a/package.json
+++ b/package.json
@@ -18,7 +18,7 @@
     "tap": "^14.6.9"
   },
   "dependencies": {
-    "minipass": "^3.0.0"
+    "minipass": "^7.0.0"
   },
   "files": [
     "index.js"
